chain/chain/src/test_utils/kv_runtime.rs

```diff
@@ -1031,24 +1030,15 @@ impl RuntimeAdapter for KeyValueRuntime {
 
     fn apply_transactions(
         &self,
-        shard_id: ShardId,
         storage_config: RuntimeStorageConfig,
-        height: BlockHeight,
-        _block_timestamp: u64,
-        _prev_block_hash: &CryptoHash,
-        block_hash: &CryptoHash,
+        chunk: crate::types::ApplyTransactionsChunkContext,
```

Comment:
nit: can have a `use crate::types::ApplyTransactionsChunkContext` here instead of fully qualifying